pkg/commands/compute/build.go

```diff
@@ -50,11 +51,12 @@ type BuildCommand struct {
 func NewBuildCommand(parent cmd.Registerer, g *global.Data, m manifest.Data) *BuildCommand {
 	var c BuildCommand
 	c.Globals = g
-	c.Manifest = m
+	c.Manifest = m // TODO: Stop passing a non-mutable 'copy' in any commands.
 	c.CmdClause = parent.Command("build", "Build a Compute@Edge package locally")
 
 	// NOTE: when updating these flags, be sure to update the composite commands:
 	// `compute publish` and `compute serve`.
+	c.CmdClause.Flag("dir", "Project directory to build (default: current directory)").StringVar(&c.Flags.Dir)
```

Comment:
Other projects, such as git and Go, use `-C` when changing the relative directory. Wondering if we should/could include that as an alias for these flags?

Reply:
Oh nice, didn't know `-C` was a convention 👍🏻

Reply:
`Make` also uses -C for this 👍 

```
-C dir, --directory=dir
            Change to directory dir before reading the makefiles or doing
            anything else.  If multiple -C options are specified, each is
            interpreted relative to the previous one: -C / -C etc is
            equivalent to -C /etc.  This is typically used with recursive
            invocations of make.
```